common/thrift_router.h

```diff
@@ -207,6 +207,31 @@ class ThriftRouter {
     return itor->second.shard_to_hosts.size();
   }
 
+  // Note: This function is AZ unaware. It returns the total number hosts
+  // within the segment and shard.
```

Comment:
nit: can you follow the similar comment style as other functions in the file?
using:
/*
 * 
 */